store/timescale.go

```diff
@@ -0,0 +1,37 @@
+package store
+
+import (
+	"database/sql"
+
+	"github.com/jackc/pgx/stdlib"
+	"github.com/pkg/errors"
+	"github.com/replicatedcom/saaskit/tracing/datadog"
+)
+
+var TimescaleDB *sql.DB
+
+type TimescaleOpts struct {
+	URI string
+}
+
+func InitTimescale(opts TimescaleOpts) error {
+	if opts.URI == "" {
+		return errors.New("Timescale URI is not set")
+	}
+
+	RegisterDatadogDriver("pgx", &stdlib.Driver{}, "timescale")
+	db, err := datadog.OpenSQL("pgx", opts.URI)
+	if err != nil {
+		return errors.Wrap(err, "open pgx")
+	}
+
+	TimescaleDB = db
+	return nil
+}
+
+func MustGetTimescaleSession() *sql.DB {
+	if TimescaleDB == nil {
+		panic("Timescale is not initilized")
```

Comment:
I assuming if  `TimescaleDB == nil` it should try and call `InitTimescale` and then panic if it `InitTimescale` returns an error?

Reply:
This actually resolves concurrency issues.  Once mysql is moved here as well, then the mutex in datadog function will no longer be needed.